packages/eslint-config-react-native-community/index.js

```diff
@@ -24,6 +24,14 @@ module.exports = {
     'jest',
   ],
 
+  overrides: {
+    files: ['**/__tests__/**/*.js', '**/?(*.)(spec|test).js'],
```

Comment:
Should include TS and TSX files as well

Reply:
@thymikee Are you sure that's something we want to add? TSEslint configuration on the project side will be required anyway. It seems logical to keep it on the project side for the projects that use TS.

Reply:
Eslint now supports TS (or is planning to) so it makes sense.

Reply:
Wouldn't we also need to add the parser as well? Otherwise, ESLint will fail to parse no?

Reference https://github.com/facebook/jest/blob/master/.eslintrc.js#L15